### ISSUE TITLE
Emarsys::SaveEmailSignup#perform raises undefined method `utc' for nil:NilClass

### ISSUE THREAD

**Issue by ryaan-anthony** [workarea-commerce/workarea-emarsys#2]
Escher no longer provides a fallback for current_time:
https://github.com/emartech/escher-ruby/blob/master/lib/escher/auth.rb#L9

vs previous: https://github.com/emartech/escher-ruby/blob/v1.0.0/lib/escher/auth.rb#L9

workarea-emarsys-1.0.1
escher-1.0.2

**Comment by ryaan-anthony** [comment]
backtrace:

```
/workarea/vendored_gems/ruby/2.4.0/gems/escher-1.0.2/lib/escher/auth.rb:266:in `long_date'
/workarea/vendored_gems/ruby/2.4.0/gems/escher-1.0.2/lib/escher/auth.rb:206:in `format_date_for_header'
/workarea/vendored_gems/ruby/2.4.0/gems/escher-1.0.2/lib/escher/auth.rb:25:in `sign!'
/workarea/vendored_gems/ruby/2.4.0/gems/workarea-emarsys-1.0.1/app/lib/workarea/emarsys/gateway.rb:55:in `sign_request'
/workarea/vendored_gems/ruby/2.4.0/gems/workarea-emarsys-1.0.1/app/lib/workarea/emarsys/gateway.rb:19:in `create_contact'
/workarea/vendored_gems/ruby/2.4.0/gems/workarea-emarsys-1.0.1/app/workers/workarea/emarsys/save_email_signup.rb:25:in `perform'
```

**Comment by meowsus** [comment]
This has been resolved with #3. Closing.
